src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.html

```diff
@@ -423,133 +388,198 @@
+                          for="grantee"
+                          i18n>Grantee
+                      <cd-helper>Select a grantee (user group) to modify it's permisions</cd-helper>
                     </span>
-                  <select id="aclPermission"
-                          name="aclPermission"
-                          class="form-input form-select"
-                          formControlName="aclPermission">
-                    <option *ngFor="let permission of aclPermissions"
-                            [value]="permission"
-                            i18n>{{ permission }}</option>
-                  </select>
-                  <span class="invalid-feedback"
-                        *ngIf="bucketForm.showError('aclPermission', frm, 'required')"
-                        i18n>This field is required.</span>
+                    <select id="grantee"
+                            name="grantee"
+                            class="form-input form-select"
+                            formControlName="grantee"
+                            (change)="onSelectionFilter()">
+                      <option *ngFor="let item of grantees"
+                              [value]="item"
+                              i18n>{{ item }}</option>
+                    </select>
+                    <span class="invalid-feedback"
+                          *ngIf="bucketForm.showError('grantee', frm, 'required')"
+                          i18n>This field is required.</span>
+                    <span class="input-group-text"
+                          for="aclPermission"
+                          i18n>Permissions
+                      <cd-helper>Select the permision to give to the selected grantee.
+                          Regardless, the owner of the bucket will always have
+                            FULL CONTROL access</cd-helper>
+                      </span>
+                    <select id="aclPermission"
+                            name="aclPermission"
+                            class="form-input form-select"
+                            formControlName="aclPermission">
+                      <option *ngFor="let permission of aclPermissions"
+                              [value]="permission"
+                              i18n>{{ permission }}</option>
+                    </select>
+                    <span class="invalid-feedback"
+                          *ngIf="bucketForm.showError('aclPermission', frm, 'required')"
+                          i18n>This field is required.</span>
+                  </div>
                 </div>
               </div>
             </div>
           </div>
+        </fieldset>
+
+        <!-- Advanced -->
+        <div class="row"
+             *ngIf="!editing">
+          <div class="col-sm-12">
+            <a class="float-end mt-2"
+               (click)="advancedEnabled = true; false"
+               *ngIf="!advancedEnabled"
+               href=""
+               i18n>Advanced Settings</a>
```

Comment:
cc @ceph/dashboard need your inputs on usage on this naming.
Does it makes sense or I change it back to "Advanced..." as in RBD image form:

https://github.com/ceph/ceph/blob/6c9d1b64849f1531cde0b6dd9bc1fd91c9ce153e/src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.html#L308

thanks!